components/NFTMint/WriterMessage.vue

```diff
@@ -202,6 +217,20 @@ export default class WriterMessage extends Vue {
     this.$emit('update-mint-amount', mintAmount)
   }
 
+  resetCollectableBeforeDate(value: Event) {
+    const { checked } = (value.target as HTMLInputElement)
+    if (!checked) {
+      this.$emit('update:collectableBeforeDate', '')
+      this.$emit('update-collectable-before-date', '')
+    }
```

Comment:
Would this break if I uncheck then check the checkbox? A date input would be shown, but no `collectable-before-date` event is fired since no @change has occurred yet

Reply:
The field would be reset if the checkbox is unchecked:

https://github.com/likecoin/app-like-co/assets/33746295/b76aa335-73ff-442f-8a38-fc1043cf680f

Is this behavior works OK?

Reply:
The above actions didn't click the background actually, I think the `computed` did the job, and the Vue extension UI has a little latency.